x-pack/packages/security-solution/features/src/cases/v1_features/kibana_features.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+import { DEFAULT_APP_CATEGORIES } from '@kbn/core-application-common';
+import { KibanaFeatureScope } from '@kbn/features-plugin/common';
+import type { BaseKibanaFeatureConfig } from '../../types';
+import { APP_ID, CASES_FEATURE_ID, CASES_FEATURE_ID_V2 } from '../../constants';
+import type { CasesFeatureParams } from '../types';
+
+/**
+ * @deprecated Use getCasesBaseKibanaFeatureV2 instead
+ */
+export const getCasesBaseKibanaFeature = ({
+  uiCapabilities,
+  apiTags,
+  savedObjects,
+}: CasesFeatureParams): BaseKibanaFeatureConfig => {
+  return {
+    deprecated: {
+      // TODO: Add docLinks to link to documentation about the deprecation
+      notice: i18n.translate(
+        'securitySolutionPackages.features.featureRegistry.linkSecuritySolutionCase.deprecationMessage',
+        {
+          defaultMessage:
+            'The {currentId} permissions are deprecated, please see {casesFeatureIdV2}.',
+          values: {
+            currentId: CASES_FEATURE_ID,
+            casesFeatureIdV2: CASES_FEATURE_ID_V2,
+          },
+        }
+      ),
+    },
+    id: CASES_FEATURE_ID,
+    name: i18n.translate(
+      'securitySolutionPackages.features.featureRegistry.linkSecuritySolutionCaseTitleDeprecated',
+      {
+        defaultMessage: 'Cases (Deprecated)',
+      }
+    ),
+    order: 1100,
+    category: DEFAULT_APP_CATEGORIES.security,
+    scope: [KibanaFeatureScope.Spaces, KibanaFeatureScope.Security],
+    app: [CASES_FEATURE_ID, 'kibana'],
+    catalogue: [APP_ID],
+    cases: [APP_ID],
+    privileges: {
+      all: {
+        api: apiTags.all,
+        app: [CASES_FEATURE_ID, 'kibana'],
+        catalogue: [APP_ID],
+        cases: {
+          create: [APP_ID],
+          read: [APP_ID],
+          update: [APP_ID],
+        },
+        savedObject: {
+          all: [...savedObjects.files],
+          read: [...savedObjects.files],
+        },
+        ui: uiCapabilities.all,
+        replacedBy: [
+          {
+            feature: CASES_FEATURE_ID_V2,
+            privileges: ['minimal_all', 'create_comment', 'case_reopen'],
```

Comment:
Please see: https://github.com/elastic/kibana/pull/186800/files/de7a1d15b0c0cccd00b1d8595b9c25142af5dcd6#diff-d887981d43bbe30cda039340b906b0fa7649ba80230be4de8eda326036f10f6fR32-R35

Users currently with 'all` have the ability to `create_comment` and `case_reopen`. Since these are being broken out into sub-privileges, they need to be provided here so when a customer that has modified the sub features upgrade upgrades, they will still retain the same behavior

For more on `minimal_all`: https://github.com/elastic/kibana/blob/main/x-pack/packages/security/authorization_core_common/src/privileges/minimal_privileges.ts#L9-L23